pandas/window.pyx

```diff
@@ -96,757 +121,1281 @@ def _check_minp(win, minp, N, floor=1):
 # Physical description: 366 p.
 #               Series: Prentice-Hall Series in Automatic Computation
 
-#-------------------------------------------------------------------------------
-# Rolling sum
+# ----------------------------------------------------------------------
+# The indexer objects for rolling
+# These define start/end indexers to compute offsets
+
+
 @cython.boundscheck(False)
 @cython.wraparound(False)
```

Comment:
Any reason not to disable these globally at the Cython module level?

`# cython: boundscheck=False`